extensions/wikia/Email/i18n/FounderDigest.i18n.php

```diff
@@ -0,0 +1,35 @@
+<?php
+$messages = array();
+
+$messages['en'] = array(
+	'emailext-founder-activity-digest-subject' => 'The latest activity on {{SITENAME}}',
+	'emailext-founder-views-digest-subject' => "Today's views on {{SITENAME}}",
+	'emailext-founder-digest-summary' => "It's time for your daily dose of activity from {{SITENAME}}.",
+	'emailext-founder-activity-digest-link-label' => 'Recent Wikia Activity',
+	'emailext-founder-activity-digest-footer' => 'Get more tips on building your wikia and ask questions on [$1 Community Central].',
```

Comment:
Garth constructed a similar link in the last email ticket he worked on. Apparently we have to hardcode these community URLS and have the translators do the same in order to have them localized. [Here's](https://github.com/Wikia/app/pull/7585/files#diff-239e1d7b95191079f1350ff59d25599fR18) the discussion about it.